### PR TITLE
Feature/seed 739 language direction

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp please review

File: iogt/templates/base.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 {% get_settings %}
 
 <!DOCTYPE html>
+{% if languages %}
```
Comment (peterp) [inline iogt/templates/base.html]: please remove if statement.

**Comment by Mitso** [comment]
@peterp please review

screenshot 2016-07-29 13 50 41

File: iogt/static/css/_base.scss [hunk 2]
```diff
@@ -83,4 +83,5 @@ input[type="submit"] {
 	width: 100%;
 	padding: 10px 0;
 	text-align: left;
+	white-space: normal;
```
Comment (peterp) [inline iogt/static/css/_base.scss]: you wouldn't have to change this, as `normal` is the default.

File: iogt/templates/base.html [hunk 3]
```diff
@@ -22,13 +22,24 @@
         <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        {% get_current_language as LANGUAGE_CODE %}
         {% compress css %}
-        <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
+<<<<<<< HEAD
```
Comment (peterp) [inline iogt/templates/base.html]: Whoops, looks like we have a merge conflict here.

File: iogt/templates/base.html [hunk 4]
```diff
@@ -90,17 +102,17 @@
...
             {% endfor %}
           </ul>
         </div>
-      </div>
+      </div>  
       {% endif %}
 
       <div class="header">
-        <a href="/" >
+        <a href="/" class="logo-link">
```
Comment (peterp) [inline iogt/templates/base.html]: can we call this `logo` instead of `logo-link`, and remove it from the image?

File: iogt/templates/base.html [hunk 5]
```diff
@@ -22,13 +22,24 @@
...
-        <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
+<<<<<<< HEAD
++           <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss'%}">
+           {% if LANGUAGE_CODE|language_bidi == True %}
+             <link rel="stylesheet" type="text/x-scss" href="{% static 'css/rtl-app.scss'%}">
+           {% endif %}
+=======
+            <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss'%}">
+          {% if LANGUAGE_CODE|language_bidi == True %}
+            <link rel="stylesheet" type="text/x-scss" href="{% static 'css/rtl-app.scss'%}">
```
Comment (peterp) [inline iogt/templates/base.html]: shouldn't the majority of the CSS changes be in this file? `rtl-app.scss?` also, can we rename this to `style_rtl.scss`

**Comment by Mitso** [comment]
@peterp please review

File: iogt/static/css/modules/_modules.scss [hunk 6]
```diff
@@ -1,10 +1,13 @@
 
-.logo {
+.logo  {
+	overflow: auto;
```
Comment (peterp) [inline iogt/static/css/modules/_modules.scss]: is this required?

File: iogt/static/css/modules/_m-section.scss [hunk 7]
```diff
@@ -10,19 +10,20 @@
...
 .section-list-nav {
 	padding: 10px 15px;
 	border-top: solid 1px #e4e4e4;
 	text-transform: uppercase;
 	font-weight: bold;
+	overflow: auto;
 }
 .section-list-nav__item {
 	display: inline;
+	text-align: left;
```
Comment (peterp) [inline iogt/static/css/modules/_m-section.scss]: isn't text-align left by default?

File: iogt/static/css/modules/_m-section.scss [hunk 8]
```diff
@@ -10,19 +10,20 @@
...
 }
 
-
-
 .section-list-nav {
 	padding: 10px 15px;
 	border-top: solid 1px #e4e4e4;
 	text-transform: uppercase;
 	font-weight: bold;
+	overflow: auto;
```
Comment (peterp) [inline iogt/static/css/modules/_m-section.scss]: is this required?

File: iogt/templates/base.html [hunk 9]
```diff
@@ -22,13 +22,17 @@
         <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        {% get_current_language as LANGUAGE_CODE %}
         {% compress css %}
-        <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
+          <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss'%}">
+          {% if LANGUAGE_CODE|language_bidi == True %}
+          <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style-rtl.scss'%}">
```
Comment (peterp) [inline iogt/templates/base.html:29]: where is this file?

**Comment by Mitso** [comment]
@peterp thanks for the review.

I've removed those styles, they were left over from the class based direction setup I think.

**Comment by peterp** [comment]
👍

**Comment by peterp** [comment]
🍰

**Pull request merged**